windows_install_dependencies.bat

```diff
@@ -0,0 +1,56 @@
+::
```

Comment:
Is this supposed to be executed in CI only or also by the user? If it's for the user as well, perhaps we should include it in the docs along with the information that `wget` and `tar` are needed for this to work.

Reply:
I will update the documentation once the script installs all of the dependencies, currently lets stick to manual install instructions.